src/ctia/auth.clj

```diff
@@ -68,6 +68,11 @@
     ;; Verdict
     :read-verdict
 
+    ;; Bundle
+    :create-bundle
+    :read-bundle
+    :delete-bundle
+
```

Comment:
I'm not sure, this entity can hold huge records and impact performance, I would make it so that a restricted set of specific applications are allowed to use it.